apps/synapse-oauth-signin/src/style/theme.ts

```diff
@@ -149,6 +150,7 @@ const themeObject = {
     sectionTitle: {},
     subsectionHeader: {},
     dataFieldKey: {},
+    monospace: {},
```

Comment:
Fix build by including new custom typography variant that was added since this was last changed